extractor/src/main/java/org/schabi/newpipe/extractor/localization/TimeAgoParser.java

```diff
@@ -93,7 +93,7 @@ private boolean textualDateMatches(final String textualDate, final String agoPhr
         final String escapedSeparator = patternsHolder.wordSeparator().equals(" ")
                 // From JDK8 → \h - Treat horizontal spaces as a normal one
                 // (non-breaking space, thin space, etc.)
-                ? "[ \\t\\xA0\\u1680\\u180e\\u2000-\\u200a\\u202f\\u205f\\u3000]"
```

Comment:
Why is `\d` added here? Could you update the above comment to indicate this?

Reply:
To be able to parse strings where numbers directly follow digit (eg. `1wk`. I'll add that to the comment